cppcache/integration/test/CqPlusAuthInitializeTest.cpp

```diff
@@ -64,8 +64,7 @@ const int32_t CQ_PLUS_AUTH_TEST_REGION_ENTRY_COUNT = 50000;
 
 Cache createCache(std::shared_ptr<SimpleAuthInitialize> auth) {
   auto cache = CacheFactory()
-                   .set("log-level", "debug")
-                   .set("log-file", "geode_native.log")
+                   .set("log-level", "none")
```

Comment:
Thank you for this!!!!

Reply:
OMG I couldn't figure out why this stupid test was timing out in my pipeline, then noticed _a test that does 50k puts_ was logging at `debug` level.  Good grief.  So yeah, I set all the tests to `none`, except there's one that _is_ slurping logs that I had to leave at `debug` :(.